bespin_lando_k8s/tasks/create-services.yml

```diff
@@ -0,0 +1,26 @@
+- name: Setup lando config
+  template:
+    src: lando_k8s_config.yml.j2
+    dest: "{{ lando_config_path }}"
+
+- name: Create lando kubernetes job runner container
+  docker_container:
+    image: "{{ bespin_settings.lando.build.image_name }}:{{ bespin_settings.lando.build.version }}"
+    name: "bespin-lando-k8s-{{ k8s_cluster_name }}"
+    volumes:
+      - "{{ lando_config_path }}:{{ lando_config_path }}"
+    etc_hosts: "{{ bespin_hosts_list }}"
+    state: started
+    restart_policy: always
+    command: python -m lando.k8s.lando "{{ lando_config_path }}"
+
+- name: Create lando kubernetes watcher container
+  docker_container:
+    image: "{{ bespin_settings.lando.build.image_name }}:{{ bespin_settings.lando.build.version }}"
+    name: "bespin-watcher-k8s-{{ k8s_cluster_name }}"
+    volumes:
+      - "{{ lando_config_path }}:{{ lando_config_path }}"
```

Comment:
I know this was an existing line moved from another file, but can the volume be read-only?